bindings/src/Capgemini.PowerApps.SpecFlowBindings/TestDriver.cs

```diff
@@ -60,6 +60,14 @@ public void InjectOnPage(string authToken)
             this.javascriptExecutor.ExecuteScript(scriptBuilder.ToString());
         }
 
+        /// <inheritdoc cref="ITestDriver"/>
+        public void UpdateAccessToken(string accessToken)
+        {
+            var script = $"window.appUserRecordRepository.updateAccessToken('{accessToken}')";
```

Comment:
I might be tempted on this one to continue to use the `driver` object as the façade and using `ExecuteDriverFunction` rather than working with the internals.

Reply:
Actually, if we go with my suggestion of making the token a method parameter then that will negate the need for this.

Reply:
Where are you picturing the method parameter? We already have `updateAccessToken('{accessToken}')` that could be `setAccessToken('{accessToken}')`?